src/screens/dashboard/components/Notification.tsx

```diff
@@ -54,34 +65,36 @@ const Notification = ({ source: author, wasSeenByHolder, type, ...p }: Notificat
 
   return (
     <>
-      <BaseOpacity
-        activeOpacity={1}
-        onPress={onPress}
-        opacity={opacity}
+      <Box
         backgroundColor="lightGrey"
         borderRadius="lmin"
         marginVertical="s"
         marginTop="none"
         height={88}
-        flexDirection="row"
         overflow="hidden">
-        <NotificationThumbnail author={author} type={type} />
-        <Box flex={1}>
-          <NotificationContent
-            endDate={endDate}
-            type={type}
-            description={description}
-            firstName={author.firstName}
-            lastName={author.lastName}
-            isSeen={wasSeenByHolder}
-          />
-          <Box marginBottom="s" marginRight="m" alignSelf="flex-end">
-            <Text variant="textXS" color="darkGreyBrighter">
-              {formatDate(new Date(p.createdAt), 'ago')}
-            </Text>
+        <BaseOpacity
+          activeOpacity={defaultOpacity}
+          onPress={onPress}
+          opacity={opacity}
+          flexDirection="row">
+          <NotificationThumbnail author={author} type={type} />
+          <Box flex={1}>
+            <NotificationContent
+              endDate={endDate}
+              type={type}
+              description={description}
+              firstName={author.firstName}
+              lastName={author.lastName}
+              isSeen={wasSeenByHolder}
+            />
+            <Box marginBottom="s" marginRight="m" alignSelf="flex-end">
+              <Text variant="textXS" color="darkGreyBrighter">
+                {formatDate(new Date(p.createdAt), 'ago')}
+              </Text>
+            </Box>
           </Box>
-        </Box>
-      </BaseOpacity>
+        </BaseOpacity>
+      </Box>
       {modalUser && (
```

Comment:
maybe its possible to move this modal component inside Box and remove fragments?